cpp/src/lists/explode.cu

```diff
@@ -187,7 +187,7 @@ std::unique_ptr<table> explode_outer(table_view const& input_table,
     });
   thrust::inclusive_scan(rmm::exec_policy(stream),
                          null_or_empty,
-                         null_or_empty + sliced_child.size(),
+                         null_or_empty + explode_col.size(),
```

Comment:
I was going to make this suggestion. This reads so much better. (The slicing is incidental to the context of this function.)

Reply:
That is confusing at first, but what's happening here is we start with offset size, which is `explode_col.size()+1` as you noted and then subtract one for the offset of the values and then subtract another because we're building this comparing to the next one, so at the end we won't have a next one to compare.